hasher-matcher-actioner/webapp/src/pages/settings/ThreatExchangeSettingsTab.jsx

```diff
@@ -70,16 +70,22 @@ export default function ThreatExchangeSettingsTab() {
         setShowToast(true);
       });
   };
+
+  const refreshDatasets = () => {
+    fetchAllDatasets(setLoading(false)).then(response => {
```

Comment:
I'm confused by this line, `setLoading(false)` as a parameter for `fetchAllDatasets.` 
When is it called?

Reply:
yeah I think that's wrong, it should not be a parameter, I should call ```setLoading(false)``` before calling ```fetchAllDatasets```, I made the change